dist/_locales/ko/messages.json

```diff
@@ -0,0 +1,222 @@
+{
+  "app_name": {
+    "message": "TabTabTab - Tab Manager",
+    "description": "The name of the application. Shown in the web store."
+  },
+  "app_name_short": {
+    "message": "TabTabTab",
+    "description": "The short name of the application. Shown at the top of the dropdown menu."
+  },
+  "app_desc": {
+    "message": "브라우저 탭 관리를 도와주는 확장 프로그램입니다.",
+    "description": "The short description of the application. Shown in the web store."
+  },
+  "window": {
+    "message": "Window",
+    "description": ""
+  },
```

Comment:
Could you translate this part too? Thank you!

Reply:
Done!